test/integration/targets/connection_aws_ssm/aws_ssm_integration_test_setup_teardown/tasks/main.yml

```diff
@@ -0,0 +1,113 @@
+---
+# tasks file for ec2
+
+- name: Install Session Manager Plugin for debian
+  include_tasks: debian.yml
+  when: ansible_distribution == "Ubuntu" or ansible_distribution == "Debian"
+  tags: setup_infra
+
+- name: Install Session Manager Plugin for RedHat
+  include_tasks: redhat.yml
+  when: ansible_distribution == "CentOS" or ansible_distribution == "Red Hat Enterprise Linux"
```

Comment:
Please verify Amazon Linux Distro as well.

Reply:
Tested and Updated ansible_distro for Amazon linux